apps/system/utils/kdbg_cpuload.c

```diff
@@ -19,17 +19,23 @@
  * Included Files
  ****************************************************************************/
 #include <tinyara/config.h>
+#include <sched.h>
 #include <stdio.h>
 #include <string.h>
-#include <sched.h>
+#include <stdlib.h>
 #include <stdint.h>
 #include <stdbool.h>
+#include <dirent.h>
 #ifdef CONFIG_ENABLE_CPULOAD_MONITOR
```

Comment:
Is it possible to compile this file without enabling this config?

Reply:
yes. I checked